chapters/revisions.tex

```diff
@@ -1428,7 +1428,7 @@ \subsection{Main changes in Modelica 3.1}\doublelabel{main-changes-in-modelica-3
 
 \begin{itemize}
 \item
-  Overloading of operators like `+' or `*' to allow convenient usage of
+  Overloading of operators like \lstinline!+! or \lstinline!*! to allow convenient usage of
```

Comment:
Similarly: \lstinline!'+'! or \lstinline!'*'!

Reply:
OK, I'm not convinced that it really isn't just `+` that is being overloaded (and that the name `'+'` is just the name we use when defining the overloading), but I changed it as suggested anyway.